docs/user/05-60-kyma-bindings.md

```diff
@@ -19,5 +19,5 @@ The Broker returns a kubeconfig file in the response body. The kubeconfig file c
 
 | Name                   | Default | Description                                                                                                                                                                                                                                                                                                                                                          |
 |------------------------|---------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| **token_request**      | `false` | If set to `true`, the Broker returns a kubeconfig with a JWT token used as a user authentication mechanism. The token is generated using Kubernetes TokenRequest attached to a ServiceAccount, ClusterRole, and ClusterRoleBinding, all named `kyma-binding-{{binding_id}}`. Such an approach allows for easily modifying the permissions granted to the kubeconfig. |
-| **expiration_seconds** | `600`   | Specifies the duration (in seconds) for which the generated kubeconfig is valid. If not provided, the default value of `600` seconds (10 minutes) is used.                                                                                                                                                                                                             |
\ No newline at end of file
+| **token_request**      | `false` | If set to `true`, the Broker returns a kubeconfig with a JWT token used as a user authentication mechanism. The token is generated using Kubernetes TokenRequest attached to a ServiceAccount, ClusterRole, and ClusterRoleBinding, all named `kyma-binding-{{binding_id}}`. Such an approach allows for easily modifying the permissions granted to the kubeconfig.  Default value (false) generates the kubeconfig using Gardener's `shoots/adminkubeconfig`. |
+| **expiration_seconds** | `600`   | Specifies the duration (in seconds) for which the generated kubeconfig is valid. If not provided, the default value of `600` seconds (10 minutes) is used. |
```

Comment:
This is duration of kubeconfig or duration of token? Or duration of both?

Reply:
There is not difference because when token expires Kubeconfig is useless.

Reply:
We have kubeconfig validation functions in KEB, so the term valid/invalid refers to syntax, not to usefulness of data.

Reply:
@jaroslaw-pieszka I'm not sure how to phrase it better, because it's a user facing documentation and from the user's point of view, after the token expires, the kubeconfig is invalid for him.